### ISSUE TITLE
new pg gem v.1.0.0 incompatible with the rails

### ISSUE THREAD

**Issue by zoonoo** [rails/rails#31684]
Hello all, it seems that the new version of [pg gem for postgres](https://rubygems.org/gems/pg/) has been released a few days ago. After updating my gem from 0.21.0 to 1.0.0, I get the following error.

```
/home/vagrant/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.4/lib/active_record/
connection_adapters/connection_specification.rb:188:
in 'rescue in spec': Specified 'postgresql' for database adapter, 
but the gem is not loaded. Add `gem 'pg'` to your Gemfile 
(and ensure its version is at the minimum required by ActiveRecord). (Gem::LoadError)
```

### System configuration
**Rails version**: 5.1.4

**Ruby version**: 2.3.3

**Comment by VitaliyAdamkov** [comment]
`cat ~/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activerecord-5.1.4/lib/active_record/connection_adapters/postgresql_adapter.rb|head -3`

> \# Make sure we're using pg high enough for type casts and Ruby 2.2+ compatibility
> gem "pg", "~> 0.18"
> require "pg"

**Comment by nbulaj** [comment]
Fixed in 181836d894b0a198b150583fe4ac97dc70badcea
